shared/chat/conversation/attachment-fullscreen/container.tsx

```diff
@@ -12,21 +12,27 @@ import {imgMaxWidthRaw} from '../messages/attachment/image/image-render'
 
 const blankMessage = Constants.makeMessageAttachment({})
 
-type OwnProps = Container.RouteProps<{message: Types.MessageAttachment}>
+type OwnProps = Container.RouteProps<{conversationIDKey: Types.ConversationIDKey; ordinal: Types.Ordinal}>
 
 const Connected = (props: OwnProps) => {
-  const [m, setMessage] = React.useState(Container.getRouteProps(props, 'message', blankMessage))
+  const conversationIDKey = Container.getRouteProps(props, 'conversationIDKey', Constants.noConversationIDKey)
+  const inOrdinal = Container.getRouteProps(props, 'ordinal', 0)
+  const state = Container.useSelector(s => s)
```

Comment:
this thrashes every time

Reply:
That is existing code, I just moved it.

Reply:
https://github.com/keybase/client/blob/master/shared/chat/conversation/attachment-fullscreen/container.tsx#L21

Reply:
sure, its a 1 liner to fix

Reply:
It's not, I pass the whole thing to some constants function now. Why does it matter for a fullscreen popup?